packages/fastify/src/withClerkMiddleware.ts

```diff
@@ -43,10 +41,8 @@ export const withClerkMiddleware = (options: ClerkFastifyOptions) => {
     }
 
     if (requestState.isInterstitial) {
-      const interstitialHtmlPage = clerkClient.localInterstitial({
-        publishableKey,
-        frontendApi: constants.FRONTEND_API,
-      });
+      // TODO(@dimkl): use empty string for frontendApi until type is fixed in @clerk/backend to drop it
+      const interstitialHtmlPage = clerkClient.localInterstitial({ publishableKey, frontendApi: '' });
```

Comment:
❓ oh is this required ? 😅

Reply:
yes. Will be fixed in the `@clerk/backend` deprecations PR